bin/www

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
```

Comment:
Just out of curiosity, why don't you prefix this file with .js, and instead specify it as a node executable or whatever you make it in a she-bang?